controllers/users-controller.js

```diff
@@ -0,0 +1,5 @@
+const { getAllTopics } = require('../models/topics')
+
+exports.getTopics = (req,res) => {
```

Comment:
I think there may be a little confusion here Lucca as there is a `getTopics` function in both your topics controllers file and users also

I would still like to suggest instead of `res.send(data)` we apply it to an object with an appropriate key `res.send({ topics: data })`